vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -694,6 +694,8 @@ void AdsImpl::OnPageLoaded(
     return;
   }
 
+  CheckAdConversion(url);
```

Comment:
`CheckAdConversion` should be called for every call to `OnPageLoaded` otherwise ad conversion detection failed if the target URL of the last shown ad matches the conversion URL pattern